app/models/concerns/profile_searching.rb

```diff
@@ -60,6 +76,12 @@ def test_results?(_filter, _operator, value)
       { conditions: profiles.arel.where_sql.gsub(/^where /i, '') }
     end
 
+    def policy_test_results?(_filter, _operator, value)
+      has_test_results = ActiveModel::Type::Boolean.new.cast(value)
```

Comment:
Same nitpick here: `s/has_test_results/has_policy_test_results/`

Reply:
That shadows the scope. ;-)